mob

```diff
@@ -219,18 +219,19 @@ if 'dockerfile' in mobconf['image']:
     with open(os.path.join(top_level, docker_ver_file), 'r') as file:
         version_data = file.read().strip()
 elif 'Dockerfile-version' in mobconf['image']:
+    docker_dir = None
     version_data = mobconf['image']['Dockerfile-version']
 else:
     raise Exception("Neither 'dockerfile' nor 'Dockerfile-version' was found in .mobconf")
 
 # Parse what version data we found
 ver_segments = version_data.split('_')
 if len(ver_segments) != 2:
-    raise Exception("Expected form A_B for Dockerfile-version, found '" + data + "' which parsed as: " + str(ver_segments))
+    raise Exception("Expected form A_B for Dockerfile-version, found '" + version_data + "' which parsed as: " + str(ver_segments))
```

Comment:
This should use string formatting. Either 'bleh bleh {} bleh'.format(version_data, ver_segments) or if only supporting Python>3.7, `f'bleh bleh {version_data}'`

Reply:
The rest of the file tends to use string concatenation, so I think if we want to improve it, we should do a pass at the whole file to keep it consistent.